exe/kubernetes-deploy

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+require 'kubernetes-deploy'
+
+options = {
+  namespace: ARGV[0],
+  context: ARGV[1],
+  environment: ENV['ENVIRONMENT'],
+  current_sha: ENV['REVISION'],
+  kubeconfig_path: ENV["KUBECONFIG"],
+}
+
+options[:template_dir] = ENV['K8S_TEMPLATE_FOLDER'] || File.expand_path("./config/deploy/#{options.fetch(:environment)}")
```

Comment:
I like `KUBECONFIG` if already used. `kubectl(1)` uses a combination of environment variables (for setting your environment) and arguments for whatever is very applicable to the specific command (and not multiple commands, such as the environment).